src/components/AllServicesDropdown/PlatformServicesLinks.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import ChromeLink from '../ChromeLink';
+
+const PlatformServiceslinks = () => {
+  return (
+    <>
+      <ChromeLink href="/ansible/ansible-dashboard" className="pf-v5-u-pl-lg pf-v5-u-pb-sm">
```

Comment:
The URL should be `/ansible`. Otherwise, it will do a full page refresh. The is not registered for such a pathname.